frontend/kesaseteli/employer/browser-tests/index-page/userActions.testcafe.ts

```diff
@@ -40,20 +39,3 @@ test('can change to languages', async () => {
   await languageDropdown.actions.changeLanguage('en', 'fi');
   await headerUser.expectations.loginButtonIsTranslatedAs('fi');
 });
-
```

Comment:
test is obsolete after we implemented the idea of loading the ui language from draft application. There always might be a draft application left from previous run or manually testing which can be of any langauge.